packages/fxa-settings/src/components/UnitRow/index.stories.tsx

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { useBooleanState } from 'fxa-react/lib/hooks';
+import { UnitRow } from '.';
+import { Modal } from '../Modal';
+
+storiesOf('components/UnitRow', module)
+  .add('basic, with falsey headerValue', () => (
+    <UnitRow header="Some header" headerValue={null} />
+  ))
+  .add('with CTA and falsey headerValue', () => (
```

Comment:
Yeah actually, it does make sense to do this in `UnitRow`. `UnitRowWithAvatar` needs to have `null` explicitly passed in since `avatarUrl` could be `null`.

Reply:
@jodyheavener actually, we can't do this because `displayName` can also be `null`.